_sass/constants.scss

```diff
@@ -1,9 +1,9 @@
 $IS_LTR : true;
 
-$FONT-STACK: roboto, sans-serif;
+$FONT-STACK: Noto Sans, sans-serif;
 // $FONT_BASE: 1rem;
 // $FONT_SCALE: 1.33333;
-$FONT_SIZE_N: 18px;
+$FONT_SIZE_N: 16px;
```

Comment:
Manros told me the standard font-size is 18px. Can you double check with him again?